src/client/debugger/jupyter/debugControllers.ts

```diff
@@ -57,13 +57,16 @@ export class RunByLineController implements IDebuggingDelegate {
     public continue(): void {
         if (typeof this.lastPausedThreadId !== 'number') {
             traceVerbose(`No paused thread, can't do RBL`);
+            this.stop();
```

Comment:
With this, if I click the RBL button and then click it again before the first pause, it disconnects. That's not that bad of a problem but could be annoying, especially since it will then run the whole cell, even if I didn't want/expect it to. 

I guess this is working around a case where we attach to debugpy and it doesn't respond, but if we have fixed the issue on our end of trying to attach twice at a time, then I don't understand how else we get into that case